src/modules/checkIsValidUserInput.js

```
@@ -9,7 +9,25 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput[0] === '0') {
+    return false;
+  }
+
+  if (userInput.length !== 4) {
+    return false;
```

Issue: The function returns false for invalid input, but it should throw an error with a descriptive message according to the requirements.